src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -54,6 +58,22 @@ export default View.extend({
       links = links.filter(l => $.isPlainObject(l));
     }
 
+    if (this.showFactorPageCustomLink()) {
```

Comment:
why this needed in Base class.. It's unnecessary for the views that doesn't use it..

Better find ways to override in the respective view itself.